internal/impl/devbox.go

```diff
@@ -526,15 +538,22 @@ func (d *Devbox) Services() (services.Services, error) {
 	return result, nil
 }
 
-func (d *Devbox) StartServices(ctx context.Context, serviceNames ...string) error {
-	if !d.IsEnvEnabled() {
```

Comment:
🤔 I did re-implement this function in the PathStack PR. I wonder if that update caused this bug.